app/models/repository_checklist_value.rb

```diff
@@ -18,6 +18,10 @@ def formatted
     repository_checklist_items.pluck(:data).join(' | ')
   end
 
+  def export_formatted
```

Comment:
you can  make ` | ` as variable like = `def formatted(separator = ' | ')`
and use `alias export_formatted formatted('\n')`